cli/asc.js

```diff
@@ -488,10 +468,10 @@ exports.main = function main(argv, options, callback) {
     });
   });
   let customLibDirs = [];
-  if (args.lib) {
-    let lib = args.lib;
-    if (typeof lib === "string") lib = lib.trim().split(/\s*,\s*/);
-    customLibDirs.push(...lib.map(resolveBasedir));
+  if (opts.lib) {
+    let lib = opts.lib;
+    if (typeof lib === "string") lib = lib.split(/\s*,\s*/);
+    customLibDirs.push(...lib.map(p => p.trim()));
```

Comment:
In case when every path trimmed in map we could simplify a little:
```suggestion
    if (typeof lib === "string") lib = lib.split(",");
    customLibDirs.push(...lib.map(p => p.trim()));
```